xep-0424.xml

```diff
@@ -120,9 +125,7 @@
 
   <example caption="The client sends out a retraction message"><![CDATA[
 <message type='chat' to='lord@capulet.example' id='retract-message-1'>
-  <apply-to id="origin-id-1" xmlns="urn:xmpp:fasten:0">
-    <retract xmlns='urn:xmpp:message-retract:0'/>
-  </apply-to>
+  <retract id="origin-id-1" xmlns='urn:xmpp:message-retract:1'/>
```

Comment:
I think it would be preferably if the id used is:
1. XEP-0359 origin-id or id from the message id attribute in direct chats
2. XEP-0359 stanza-id assigned by the MUC in MUCs.

As I mentioned on the mailing list, I'll write these rules into a informational XEP soonish (which once available can be referenced from here), but I'd want to make sure we update the rule at the same time we namespace bump (so we won't need another ns bump). Feel free to just copy the rules from https://xmpp.org/extensions/xep-0444.html#business-id

Reply:
Thanks, updated accordingly.